### PR TITLE
V2

### PR DESCRIPTION
**Pull request by psibre**

### CODE REVIEW

File: src/main/resources/marytts/language/dsb/abbreviations.csv [hunk 1]
```diff
@@ -0,0 +1,130 @@
+"ca.","circa"
```
Comment (psibre) [inline src/main/resources/marytts/language/dsb/abbreviations.csv:1]: We need a header row here, otherwise we cannot load the CSV...

**Comment by psibre** [comment]
We need to rebase these commits on the current master branch.

**Comment by psibre** [comment]
Compare #21, which should replace this PR.

File: src/main/resources/marytts/language/dsb/abbreviations.csv (hunk 1 above)
Comment (aStereoID) [reply src/main/resources/marytts/language/dsb/abbreviations.csv:2]: ok, just added

**Comment by psibre** [comment]
> Which of these changes fixed the Sonatype problem? Somewhere else entirely?

The Sonatype thing had to do with their infrastructure; there was an error when closing a staging repo. It's been resolved.

**Pull request merged**